tests/pmap_test.py

```diff
@@ -39,9 +40,30 @@
 from jax.config import config
 config.parse_flags_with_absl()
 
+prev_xla_flags = None
+
+# Run all tests with 8 CPU devices.
+def setUpModule():
```

Comment:
I'm wondering whether this will work in Google's test setup; you might wish to verify that before merging. Otherwise it looks ok.